R/sjTabMannWhitney.R

```diff
@@ -26,10 +26,10 @@
 #'
 #' @examples
 #' \dontrun{
-#' library(sjmisc)
-#' data(efc)
+#' #' library(sjmisc)
+#' #' data(efc)
```

Comment:
See above,  I guess my (sloppy executed)  way of testing is probably not the best one =D